documentation/add/09_test_data_storage.md

```diff
@@ -0,0 +1,94 @@
+# Test data storage, distribution and requirements for PACE project
+
+Date: 3-11-2021
+
+## Current State
```

Comment:
It is surprising that the main body of the document stops with a statement of problem and requirements, and then gives possible solution as an appendix. One might expect these latter to be part of the discussion. 

It may be useful at this point to give a brief intro:

Introduction
Data for Horace tests is currently stored in the repo, which has implications for the data size required for upload, download and storage. This document describes these issues in detail, provides requirements for alternative storage location and gives possible alternative storage strategies".

The Appendix header could then be taken out.